colour-changing-grid/style.css

```diff
@@ -1 +1,14 @@
-/* Colour changing grid CSS here */
\ No newline at end of file
+/* Colour changing grid CSS here */
+div {
```

Comment:
Not a big deal here, but you don't typically want to style a plain `div` tag in any project. Use a class instead, bonus points for a semantic name 😄